src/player.h

```diff
@@ -1177,7 +1177,7 @@ class Player final : public Creature, public Cylinder
 	std::forward_list<Party*> invitePartyList;
 	std::forward_list<uint32_t> modalWindows;
 	std::forward_list<std::string> learnedInstantSpellList;
-	std::forward_list<Condition*>
+	std::forward_list<std::shared_ptr<Condition>>
```

Comment:
Yes, of course, I am the first person who loves to have a name for everything, however I have fallen into a dilemma, which is why I have uploaded the changes as is, to receive feedback

I like: `Condition_ptr` but what name can I give to the case of `const std::shared_ptr<const Condition>`?

It really looks horrible something like this: `Constiion_ptr_const` or `Condition_const_ptr`

I will leave it up to what the team thinks and of course those people who wish to give good suggestions.

I do not have in mind to abandon the use of const, just for a simple name, I am in favor of making everything possible constness

Reply:
By using `std::shared_ptr<T>` instead of `T_ptr`, I figured we wouldn't have `Condition_ptr` and commented on the clarity, in the end, it doesn't matter